site/docs/v1/tech/identity-providers/external-jwt/index.adoc

```diff
@@ -166,20 +174,20 @@ Optionally a single key may be designated as the default key. You may use this f
 +
 In this scenario, there will be no `kid` to indicate which key was used to sign the JWT, so if a default key has been designated, the default key will be used to verify the signature.
 +
-If you will not have JWTs that do not contain the `kid` or other key identifier, you will not use this feature. When this toggle is enabled, the `Key identifier` field will be disabled and no longer required.
+If you will not have JWTs that do not contain the `kid` or another key identifier, you will not use this feature. When this toggle is enabled, the `Key identifier` field will be disabled and no longer required.
```

Comment:
I think this means:

"If you only have JWTs with the `kid` claim, or another key identifier, you will not use this feature. When this toggle is enabled, the [field]#Key identifier# field will be disabled and no longer required."